lib/db-manager.js

```diff
@@ -104,26 +104,23 @@ var dbManager = module.exports = {
     */
    updateCommitStatus: function(updatedStatus) {
       debug('Calling `updateCommitStatus` for commit %s in repo %s',
-       updatedStatus.data.commit, updatedStatus.data.repo);
+       updatedStatus.data.sha, updatedStatus.data.repo);
       var dbStatus = new DBStatus(updatedStatus);
-      var number = this.getPullNumberFromSha(dbStatus.data.repo, dbStatus.data.commit);
 
       return dbStatus.save().
-      then(function() {
-         return number;
-      }).
-      then(function(n) {
-         // `n` will be null if we are updating the build status of a commit
+      then(() => this.getPullNumberFromSha(updatedStatus.data.repo, updatedStatus.data.sha)).
+      then(function(pullNumber) {
+         // `pullNumber` will be null if we are updating the build status of a commit
          // which is no longer a pull's head commit.
-         if (n === null) {
+         if (pullNumber === null) {
             debug('updateCommitStatus: commit no longer HEAD of Pull #%s in repo %s',
-                number, updatedStatus.data.repo);
+                pullNumber, updatedStatus.data.repo);
```

Comment:
This was broken before, but the message still doesn't make a whole lot of sense (`pullNumber` is always null here). Maybe just `commit is not the HEAD of any pull, none to refresh`